src/Skill.ts

```diff
@@ -58,7 +58,7 @@ export class Skill {
 	info: Function;
 	initiatesCombat: boolean;
 	name: string;
-	options: object;
+	options: Record<string, unknown>;
```

Comment:
This is a more standard and informative way to type, `object` type is generally advised to not be used.